### PR TITLE
Fix show login view again after input credentials

### PR DESCRIPTION
**Pull request by nasli**
Related to bug (12)https://github.com/owncloud/ios/issues/923#issuecomment-347477360

### CODE REVIEW

**Comment by jesmrec** [comment]
In SAML:

1.- Attach 2 accounts
2.-  After attaching the 2nd (file list displayed), switch to the first one.

Current: both accounts with same list
Expected: Each account, its own list.

**Comment by nasli** [comment]
@jesmrec we are dealing with this bug in other branch https://github.com/owncloud/ios/pull/963

**Comment by nasli** [comment]
@jesmrec  please review issue #923 (comment)

**Comment by jesmrec** [comment]
(7) SAML -> not fixed
(12) SAML -> FIXED
(12) OAuth2 -> FIXED

@nasli

**Comment by nasli** [comment]
Ok so handle the (7) in different issue

**Comment by jesmrec** [comment]
@nasli approved. 

(7) to be handled in other issue.

File: Owncloud iOs Client/Utils/UtilsCookies.m [hunk 1]
```diff
@@ -60,45 +60,45 @@ + (void) setOnSystemCookieStorageDBCookiesOfUser:(UserDto *) user {
...
-                for (NSString *credentialKey in credentials)
-                {
-                    [credentialsStorage removeCredential:[credentials objectForKey:credentialKey] forProtectionSpace:protectionSpace];
-                }
-            }
-        }
-    }
-}
-
+//
```
Comment (pablocarmu) [inline Owncloud iOs Client/Utils/UtilsCookies.m]: Commented functions.

File: Owncloud iOs Client/Utils/UtilsCookies.h [hunk 2]
```diff
@@ -23,9 +23,9 @@
 + (void) setOnDBStorageCookiesByUser:(UserDto *) user;
 + (void) setOnSystemCookieStorageDBCookiesOfUser:(UserDto *) user;
 
-+ (void) eraseCredentialsWithURL:(NSString *)connectURL;
+//+ (void) eraseCredentialsWithURL:(NSString *)connectURL;
```
Comment (pablocarmu) [inline Owncloud iOs Client/Utils/UtilsCookies.h]: Commented code. You should erase this functions.

**Pull request merged**